test/specs/behaviors/testDefinitions.ts

```diff
@@ -1,11 +1,12 @@
 import { TestDefinition, TestMethod, TestHelper } from './testHelper'
+import * as keyboardKey from 'keyboard-key'
 
 const definitions: TestDefinition[] = []
 const testHelper = new TestHelper()
 
 // Example:  Adds attribute 'aria-pressed=true' based on the property 'active'
 definitions.push({
-  regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)' based on the property '([a-z]+)'\./g,
+  regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)' based on the property '([a-z]+)'\.$/g,
```

Comment:
why there is a separation implied for cases when only this sentence is used vs cases where this sentence can be followed by additional statements (like `This can be overriden by directly providing '([a-z -]+)' property to the component\.`)? To me the same set of tests should be run in both cases